methods/build_genome_set_from_tree_generic/display.yaml

```diff
@@ -2,10 +2,10 @@
 # Define basic display information
 #
 name     : Build Genome Set From Tree
+
 subtitle : |
```

Comment:
I'm surprised we didn't notice this earlier, but the tooltip is required so this commit is breaking the method spec:
https://narrative-dev.kbase.us/functional-site/#/narrativestore/method/build_genome_set_from_tree_generic